python-checks/src/main/java/org/sonar/python/checks/XPathCheck.java

```diff
@@ -22,15 +22,13 @@
 import com.sonar.sslr.api.AstNode;
 import com.sonar.sslr.api.Grammar;
 import org.sonar.squidbridge.checks.AbstractXPathCheck;
-import org.sonar.check.Cardinality;
 import org.sonar.check.Priority;
 import org.sonar.check.Rule;
 import org.sonar.check.RuleProperty;
 
 @Rule(
   key = "XPath",
-  priority = Priority.MAJOR,
-  cardinality = Cardinality.MULTIPLE)
```

Comment:
Removing the cardinality makes the rule a simple rule instead of a rule template.